app/src/constants/paths.js

```diff
@@ -2,3 +2,4 @@ import {normalize} from 'path'
 
 export const lib = normalize(`${__dirname}/..`)
 export const root = normalize(`${lib}/../..`)
+export const chat = 'chat'
```

Comment:
maybe just put the utility function that returns true separately and use it there locally